services/ui-src/src/components/layout/Footer.test.tsx

```diff
@@ -1,41 +1,54 @@
 import { render, screen } from "@testing-library/react";
-import { axe } from "jest-axe";
 import { RouterWrappedComponent } from "utils/testing/setupJest";
 import { Footer } from "components";
+import { testA11y } from "utils/testing/commonTests";
 
 const footerComponent = (
   <RouterWrappedComponent>
     <Footer />
   </RouterWrappedComponent>
 );
 
-describe("Test Footer", () => {
-  beforeEach(() => {
-    render(footerComponent);
-  });
+describe("<Footer />", () => {
+  describe("Test Footer", () => {
+    beforeEach(() => {
+      render(footerComponent);
+    });
 
-  test("Footer is visible", () => {
-    const footer = screen.getByRole("contentinfo");
-    expect(footer).toBeVisible();
-  });
+    test("Footer is visible", () => {
+      expect(screen.getByRole("contentinfo")).toHaveAttribute("id", "footer");
+    });
 
-  test("Logo is visible", () => {
-    expect(screen.getByAltText("HCBS logo")).toBeVisible();
-  });
+    test("Images on footer are visible", () => {
+      expect(
+        screen.getByRole("img", {
+          name: "Department of Health and Human Services, USA",
+        })
+      ).toHaveAttribute("alt", "Department of Health and Human Services, USA");
+      expect(screen.getByRole("img", { name: "HCBS logo" })).toHaveAttribute(
+        "alt",
+        "HCBS logo"
+      );
+      expect(
+        screen.getByRole("img", {
+          name: "Medicaid.gov: Keeping America Healthy",
+        })
+      ).toHaveAttribute("alt", "Medicaid.gov: Keeping America Healthy");
+    });
```

Comment:
The `name` in the `getByRole` query for images actually searches for alt text already, so the `toHaveAttribute` assertion is redundant here.

```suggestion
    test("Images on footer are visible", () => {
      expect(
        screen.getByRole("img", {
          name: "Department of Health and Human Services, USA",
        })
      ).toBeVisible();
      expect(screen.getByRole("img", { name: "HCBS logo" })).toBeVisible();
      expect(
        screen.getByRole("img", {
          name: "Medicaid.gov: Keeping America Healthy",
        })
      ).toBeVisible();
    });
```